ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -415,7 +432,7 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         if let topAccessoryView = self.topAccessoryView {
             topAccessoryView.translatesAutoresizingMaskIntoConstraints = false
 
-            let insertionIndex = contentStackView.arrangedSubviews.firstIndex(of: leadingSpacerView)! + 1
+            let insertionIndex = contentStackView.arrangedSubviews.firstIndex(of: postTitleSpacerView)! + 1
```

Comment:
How much harder would it be to add a `topAccessoryHostView` and keep that in the same spot in the `contentStackView` and just add/remove subviews from that and use `.isHidden` to hide it if necessary? It would be great if we could remove the `!`.

Reply:
I can see this being a nice-to-have, but I'd rather focus on getting this feature merged into the main branch in the short term. We can always create an issue for it later.